nengo/conftest.py

```diff
@@ -292,15 +292,15 @@ def mark_nl(nl):
 def pytest_collection_modifyitems(session, config, items):
     if config.getvalue('noexamples'):
         deselect_by_condition(
-            lambda item: getattr(item.obj, 'example', None), items, config)
+            lambda item: item.get_closest_marker("example"), items, config)
```

Comment:
I checked, and this appears in pytest 3.6.0, which is the earliest version we support anyway :+1: I ran the tests with 3.6.0, and they appear to work, with slow tests being skipped (I didn't test for other markers).